common/persistence/sql/sqlplugin/tests/context_test.go

```diff
@@ -36,7 +36,7 @@ const (
 
 func newExecutionContext() context.Context {
 	ctx := context.Background()
-	ctx, _ = context.WithTimeout(ctx, executionTimeout)
+	// ctx, _ = context.WithTimeout(ctx, executionTimeout)
```

Comment:
are you going to uncomment this before landing? or delete the line?

Reply:
good catch